attestation_verifier/Cargo.toml

```diff
@@ -17,7 +17,8 @@ fork_choice_store = { workspace = true }
 futures = { workspace = true }
 helper_functions = { workspace = true }
 itertools = { workspace = true }
-log = { workspace = true }
+tracing = { workspace = true }
```

Comment:
[**chore**](https://conventionalcomments.org/): Keep dependencies in alphabetical order